.buildkite/release-automation/verify-macos-wheels.sh

```diff
@@ -44,17 +65,23 @@ run_sanity_check() {
         cd release/util
         python sanity_check.py
     )
+    (
+        cd release/util
+        bash sanity_check_cpp.sh
+    )
     conda deactivate
 }
 
 _clean_up() {
+    bazel clean
```

Comment:
check if bazel binary exists (as the `install_bazel` might fail), if `bazel` does not exist, do not run `bazel clean`, or maybe just `bazel clean || true`

Reply:
fixed! Added `|| true` so the script doesn't exit if bazel is not found there.